syntax = "proto3";

message Author {
  int64 id = 1;
  string first = 2;
  string last = 3;
  float book_count = 4;
  float age = 5;
  string address = 6;
}

message Quote {
  int64 id = 1;
  Author author = 2;
  string content = 3;
  int64 posted_at = 4;
  string book_name = 5;
  float page_number = 6;
  float line_number = 7;
  float col_number = 8;
}

message Quotes {
    repeated Quote quotes = 1;
}
